distrobox-init

```diff
@@ -385,7 +385,7 @@ passwd --delete root
 if [ -d "/etc/skel" ]; then
 	skel_files="$(find /etc/skel/ -type f || :)"
 	for skel_file in ${skel_files}; do
-		if [ ! -f "${container_user_home}/$(basename "${skel_file}")" ]; then
+		if [ ! -f "${container_user_home}/$(basename "${skel_file}")" -a !-L "${container_user_home}/$(basename "${skel_file}")" ]; then
```

Comment:
instead of `-a` let's use `&&` to be like the rest of the code

`		if [ ! -f "${container_user_home}/$(basename "${skel_file}")"  ] && [ !-L "${container_user_home}/$(basename "${skel_file}")" ]; then`